scripts/start-mspass.sh

```diff
@@ -224,14 +260,18 @@ if [ $# -eq 0 ]; then
     tail -f /dev/null
   elif [ "$MSPASS_ROLE" = "frontend" ]; then
     start_mspass_frontend
+    if [ "$MSPASS_DB_MODE" = "shard" ]; then
+      clean_up_multiple_nodes
+    else
+      clean_up_single_node
+    fi
   else # if [ "$MSPASS_ROLE" = "all" ]
     MSPASS_DB_ADDRESS=$HOSTNAME
     MSPASS_SCHEDULER_ADDRESS=$HOSTNAME
     eval $MSPASS_SCHEDULER_CMD
     eval $MSPASS_WORKER_CMD
-    [[ -d $MONGO_DATA ]] || mkdir -p $MONGO_DATA
-    mongod --port $MONGODB_PORT --dbpath $MONGO_DATA --logpath $MONGO_LOG --bind_ip_all &
```

Comment:
@haruming Gary ran into a problem that the db was not started when using the container in all-in-one mode. I checked and found that the problem is these lines' removal. Could you please double check and fix? Thank you!